pom.xml

```diff
@@ -7,6 +7,7 @@
         <groupId>org.jamesframework</groupId>
         <artifactId>james</artifactId>
         <version>1.3-SNAPSHOT</version>
+        <relativePath>../james/james</relativePath>
```

Comment:
It is added to make the codes work on my computer, as I cloned James and James-core separately. I have added pom.xml to .gitignore. Please revise the pom.xml file accordingly.